### ISSUE TITLE
Submitting text when ICE state is disconnected not possible

### ISSUE THREAD

**Issue by lgrahl** [threema-ch/threema-web#397]
There's no need for preventing this. The SCTP association of the data channel will take care of re-sending.

**Comment by dbrgn** [comment]
But if reconnecting fails for some reason, then the text is lost without the user being aware, right?

**Comment by lgrahl** [comment]
Yes but this isn't a fix for that problem as this can (and does happen) even without ICE thinking it's disconnected. The current behaviour is just really annoying on flaky networks where ICE states switch all the time. If you want to prevent that, you will need application-level acknowledgement and cached data flow. (FYI, the latter will also be required to drop the permanent connection requirement.)

To make this clear: I would not recommend triggering any special behaviour during ICE disconnected other than displaying that information.

**Comment by dbrgn** [comment]
@lgrahl do you think we should revisit or close this issue?
